book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] Based on the requirements of this project, we shall need a method used for adding rental(add_rental) in this class. This method will permit us for renting out a particular book to a person on a particular date. [ReadMore...](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)